src/components/CldImage/CldImage.vue

```diff
@@ -7,7 +7,7 @@
 <script>
 import { Cloudinary, Transformation } from "cloudinary-core";
 import { merge, range } from "../../utils";
-import {accessibilityTransformations} from '../../constants';
+import {A11Y_TRANSFORMS, COMPONENTS, PLACEHOLDER_OPTIONS } from '../../constants';
```

Comment:
I'm not sure that changing a known term (TRANSFORMATION) to something shorter (TRANSFORMS) is helpful.

this term is confusing enough, we should try to keep it consistent.